crates/api_crud/src/community/list.rs

```diff
@@ -39,7 +38,7 @@ pub async fn list_communities(
     is_mod_or_admin: is_admin,
     ..Default::default()
   }
-  .list(&mut context.pool())
+  .list(&local_site.site, &mut context.pool())
```

Comment:
Shouldn't this be part of the `CommunityQuery`? This is different from how all the other querybuilders work.

Reply:
https://github.com/LemmyNet/lemmy/pull/4393#discussion_r1464191688

Reply:
Ah, mmk I spose that makes sense for non-optionals.